acceptance/suites/tests/010-puppetserver-cli/subcommand/ruby.rb

```diff
@@ -0,0 +1,61 @@
+require 'test/unit/assertions'
+require 'puppetserver/acceptance/gem_utils'
+
+test_name "Puppetserver 'ruby' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+additional_gem_source = ENV['GEM_SOURCE']
+# define command lines
+gem_install = "#{cli} gem install "
+if additional_gem_source
+  gem_install += " --clear-sources --source #{additional_gem_source}"
+end
+
+step "Install hiera-eyaml gem."
+on(master, "#{gem_install} hiera-eyaml")
+
+step "Check that ruby subcommand can load eyaml from hiera-eyaml"
+on(master, "#{cli} ruby -- -rrubygems -reyaml -e 'puts %{OK}'") do
```

Comment:
Did this work for you?  When running from the command line, I had to omit the "--" after ruby and use "-rhiera/backend/eyaml" instead of "-eyaml" in order to avoid a "LoadError".  Maybe would also be good to `assert_no_match` for an "error" here, too?